x/oracle/keeper/oracle.go

```diff
@@ -31,7 +31,11 @@ func (k Keeper) RegisterOracle(ctx sdk.Context, msg *types.MsgRegisterOracle) er
 
 	// TODO: Add active queue of Oracle Registration
 
-	// TODO: emit RegisterOracle event
+	ctx.EventManager().EmitEvents(sdk.Events{
```

Comment:
This function seems to be deprecated.
You can use to `ctx.EventManager().EmitTypedEvents(...)`

Reply:
Hmm.. I already knew about it. But for now, we just go for `EmitEvents` because other keepers still use `EmitEvents`.